packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/iam-role/vsphere-user-iam-role.html

```diff
@@ -0,0 +1,76 @@
+<div
+    data-wizard
+    data-wizard-on-finish=":: $ctrl.boundOnCloseClick"
+    data-wizard-on-cancel=":: $ctrl.boundOnCloseClick"
+    data-wizard-hide-cancel-button="currentStep !== 1"
+    data-wizard-hide-close-button="currentStep !== 1"
+    data-wizard-next-button-text=":: 'ua_wizard_confirm' | translate"
+    data-wizard-confirm-button-text=":: 'ua_wizard_close' | translate"
+    data-wizard-title=":: 'dedicatedCloud_USER_iam_role_title' | translate"
+    data-wizard-hide-previous-button="true"
+    data-wizard-keydown-disabled
+    data-wizard-bread-crumb
+>
+    <div data-wizard-step data-wizard-step-valid="$ctrl.formIsValid()">
+        <div class="text-center" data-ng-if="$ctrl.loaders.init">
+            <oui-spinner></oui-spinner>
+        </div>
+
+        <form data-ng-if="!$ctrl.loaders.init" name="uiamRoleForm" novalidate>
+            <p data-translate="dedicatedCloud_USER_iam_role_content"></p>
+
+            <oui-field
+                label="{{ :: 'dedicatedCloud_USER_iam_role_field_label_name' | translate }} *"
+                data-error-messages="{  'pattern': ('dedicatedCloud_USER_iam_role_name_wrong_pattern' | translate),
+                                        'roleAlreadyExist': ('dedicatedCloud_USER_iam_role_name_already_exist' | translate)
+                                    }"
+            >
+                <div class="input-group">
+                    <span
+                        class="input-group-addon"
+                        data-ng-bind="$ctrl.namePrefix"
+                    ></span>
+                    <input
+                        class="form-control"
+                        name="nameField"
+                        type="text"
+                        placeholder="{{ :: 'dedicatedCloud_USER_iam_role_role_example' | translate }}"
+                        data-ng-model="$ctrl.model.name"
+                        data-ng-pattern="$ctrl.namePattern"
+                        data-ui-validate="{ roleAlreadyExist: '!$ctrl.roleAlreadyExist($value)' }"
+                        required
+                    />
+                </div>
+            </oui-field>
+
+            <p>
+                <span>*</span>
+                <span
+                    data-translate="dedicatedCloud_USER_iam_role_field_mandatory"
+                ></span>
+            </p>
```

Comment:
This info must be displayed only if the user tries to submit the form without the entering the value.

Reply:
Doing that would be inconsistent with others view and would be bad for user experience.
As a user I want to know if the fields are required or not and which.
![Capture d’écran 2023-08-22 à 11 46 10](https://github.com/ovh/manager/assets/666182/05abea00-1bcd-4a95-946e-da9fccab0f1e)

Reply:
We dont display `*` next to the fields in manager. Can you please validate this with PO or UX team?

Reply:
Ok, the screenshot I showed you is from another modal in the Dedicated Cloud manager (dedicatedCloud_USER_import_field_mandatory)  and I use the same UI to keep consistency.
Anyway, will remove that as in my case I have only one field in the form.